src/pages/Books.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Book from '../components/Book';
+
+const Books = () => (
+  <Book />
+);
+
```

Comment:
- [ ] Great Job on making some changes so far but as required in this [previous review](https://github.com/PedroH-Neves/bookstore-react/pull/1/files#r1161157537) it seems that the changes have not been made according to the stated request changes by the previous reviewer. The current implementation only renders a single instance of the <Book /> component without any props being passed in. Kindly fix this for your project to be completed :wink: 

_note: I think to render a list of books using the <Book /> component, you will need to create an array of book objects and map over it to render each book using the <Book /> component with the appropriate props_

Reply:
hello @Alfredbis29 
I am rendering the books in the component using an array of books in the folder components at the file book.

Reply:
Hi @PedroH-Neves , Thank you for getting back to me. I appreciate your effort in rendering the books using an array of books in the Book component. However, as per our requirement, we need to render the list of books using the <Book /> component and pass the necessary props to it. as it was required in the previous review , kindly fix that then ask for another opinion :wink:

Reply:
@PedroH-Neves , Rendering a single book without props does not meet the expected behavior as mentioned in the previous review. but I think to achieve the required functionality, you can pass the array of books as a prop to the Books component, and then map over the array to render each book using the `<Book />`  component.  I hope this clarifies the requirement :upside_down_face:

Reply:
@Alfredbis29 i understand this but what i am trying to say is that the book component is just to rendering the page and not a single book

Reply:
@PedroH-Neves  check this [in the previous review](https://github.com/PedroH-Neves/bookstore-react/pull/1#discussion_r1161157537)